landing/src/styles/global.css

```
@@ -55,3 +55,108 @@
+  /* The outer frame and its pre are flex content, so both need an explicit
+     floor of zero to stay inside the blog column on narrow viewports. */
+  .expressive-code,
+  .expressive-code .frame,
+  .expressive-code .frame pre {
+    min-width: 0;
+    max-width: 100%;
+  }
+
+  .expressive-code .frame {
+    --header-border-radius: 0;
+    border-radius: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)